app/mailers/patient_mailer.rb

```diff
@@ -82,50 +58,27 @@ def assessment_sms_weblink(patient)
       url_contents = new_patient_assessment_jurisdiction_report_lang_url(p.submission_token,
                                                                          lang&.to_s || 'en',
                                                                          patient.jurisdiction.unique_identifier[0, 32])
-      account_sid = ENV['TWILLIO_API_ACCOUNT']
-      auth_token = ENV['TWILLIO_API_KEY']
-      from = ENV['TWILLIO_SENDING_NUMBER']
-      client = Twilio::REST::Client.new(account_sid, auth_token)
-      client.messages.create(
-        from: from,
-        to: Phonelib.parse(num, 'US').full_e164,
-        body: intro_contents
-      )
-      client.messages.create(
-        from: from,
-        to: Phonelib.parse(num, 'US').full_e164,
-        body: url_contents
-      )
-      add_success_history(p, patient)
+      success = TwilioSender.send_sms(patient, intro_contents)
+      success &&= TwilioSender.send_sms(patient, url_contents)
+
+      add_success_history(p, patient) if success
```

Comment:
This could also be a one-liner. I won't comment on the other places, but just generally might be able to consolidate these checks in this file.